[BUG] Change dependency assets

### Observed behavior

While trying to use this package, I am getting this error:
 ```
error NU1605: 
      Warning As Error: Detected package downgrade: Microsoft.Extensions.Logging.Abstractions from 9.0.3 to 9.0.0. Reference the package directly from the project to select a different version. 
       MyWebsite -> OpenFeature 2.3.2 -> Microsoft.Extensions.Logging.Abstractions (>= 9.0.3)
```

### Expected Behavior

The libraries are not exposed.

### How to replicate
1 - Create a new web project
2 - Reference `Microsoft.Extensions.Logging.Abstractions` version 9.0.2 or lower
3 - Add OpenFeature version 2.3.2
4 - Compile

Check attachment: [WebApplication1.zip](https://github.com/user-attachments/files/19688985/WebApplication1.zip)

--- a/Directory.Packages.props
+++ b/Directory.Packages.props
@@ -2,6 +2,10 @@
 
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <MicrosoftExtensionsVersion>8.0.0</MicrosoftExtensionsVersion>
+  </PropertyGroup>
+  <PropertyGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible($(TargetFramework), 'net9.0'))">
+    <MicrosoftExtensionsVersion>9.0.0</MicrosoftExtensionsVersion>
   </PropertyGroup>
 
   <ItemGroup Label="src">

--- a/Directory.Packages.props
+++ b/Directory.Packages.props
@@ -9,14 +9,14 @@
   </PropertyGroup>
 
   <ItemGroup Label="src">
-    <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="9.0.3" />
-    <PackageVersion Include="Microsoft.Extensions.Hosting.Abstractions" Version="9.0.3" />
-    <PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="9.0.3" />
-    <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="9.0.3" />
-    <PackageVersion Include="Microsoft.Extensions.DependencyInjection" Version="9.0.3" />
-    <PackageVersion Include="Microsoft.Extensions.Options" Version="9.0.3" />
-    <PackageVersion Include="System.Collections.Immutable" Version="9.0.3" />
-    <PackageVersion Include="System.Threading.Channels" Version="9.0.3" />
+    <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="$(MicrosoftExtensionsVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.Hosting.Abstractions" Version="$(MicrosoftExtensionsVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="$(MicrosoftExtensionsVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="$(MicrosoftExtensionsVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.DependencyInjection" Version="$(MicrosoftExtensionsVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.Options" Version="$(MicrosoftExtensionsVersion)" />
+    <PackageVersion Include="System.Collections.Immutable" Version="$(MicrosoftExtensionsVersion)" />
+    <PackageVersion Include="System.Threading.Channels" Version="$(MicrosoftExtensionsVersion)" />
     <PackageVersion Include="System.ValueTuple" Version="4.6.0" />
   </ItemGroup>
 

